spec/controller/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
```

Comment:
- [ ] Great job so far 👏🏻  However the `users_spec.rb` and `posts_spec.rb` should be  in the `spec/requests/` directory instead of spec/controller/ Please rename this 👍🏻

Reply:
Addressed